modules/op-resolver-http-config.adoc

```diff
@@ -0,0 +1,36 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: PROCEDURE
+
+[id="resolver-http-config_{context}"]
+= Configuring the HTTP resolver
+
+You can use the HTTP resolver to fetch pipelines or tasks from an HTTP URL. You can configure the default service account for the HTTP resolver by editing the TektonConfig custom resource.
+
+.Procedure
+
+. To edit the `TektonConfig` custom resource, enter the following command:
++
+[source,terminal]
+----
+$ oc edit TektonConfig config
+----
+. In the `TektonConfig` custom resource, edit the `pipeline.http-resolver-config` spec:
++
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  pipeline:
+    http-resolver-config:
```

Comment:
This is wrong, there is only https://github.com/tektoncd/pipeline/blob/main/config/resolvers/http-resolver-config.yaml#L24 as options for it.

Reply:
It should look like 
```
  # The maximum amount of time the http resolver will wait for a response from the server.
  fetch-timeout: "1m"
```